examples/adjustwindow_updateviewer_saveimage.py

```diff
@@ -0,0 +1,73 @@
+from napari.settings import get_settings
+import time
+import napari
+from napari._qt.qthreading import thread_worker
+from skimage import data
+from skimage.morphology import ball, octahedron
+import matplotlib.pyplot as plt
+
+
+def make_screenshot(viewer):
+    img = viewer.screenshot(canvas_only=True, flash=False)
+    plt.imshow(img)
+    plt.axis("off")
+    plt.show()
+
+
+get_settings().application.window_position = (900, 300) # hopefully better option to do this
+viewer = napari.Viewer()
+viewer.window.resize(600, 600) 
+#viewer.window.position(600, 600) # is it possible that this will replace get_settings().application.window_position = (900, 300)?
+
+viewer.window.qt_viewer.dockLayerControls.toggleViewAction().trigger()
```

Comment:
hmm we don't have anything else right now that will be set to be a public method like this. I wonder why this is needed though if you have `canvas_only=True` in the screenshot? I'd prefer if we could drop it, but curious what others think

Reply:
@kolibril13 and @sofroniewn. note that with #4201 this shouls all be easier!

Reply:
I changed now the syntax to 
```py
viewer.window._qt_viewer.dockLayerControls.toggleViewAction().trigger()
viewer.window._qt_viewer.dockLayerList.toggleViewAction().trigger()
```
It is not pretty, but it does what it should do.
I don't want to use `canvas_only=True` in this case, because I want to keep the aspect ratio that I've defined in `viewer.window.resize(600, 600)`.

Reply:
You can sidestepp all of this (toggling controls + setting window size) by using the new `size` argument from #4201!

Reply:
The `size` feature in `screenshot` is a really cool feature!
I currently installed  `napari 0.4.15` via conda, so I can not use this feature yet, and I don't have the time right now to set up the latest napari from GitHub.
Also, I think in this case it is better to disable the dockLayer manually, because then I have already a preview of how my screenshot will look like and there wont be any unpredictable surprises. 
I prefer the "What you see is what you get" approach here.